jenkins/jjb/templates/scenario-tests.yaml

```diff
@@ -146,4 +146,172 @@
           ENVS
 
           docker run --net grid-{version} --privileged --rm --link selenium-hub-{version}:selenium-grid-hub -v /var/run/docker.sock:/var/run/docker.sock -v /etc/docker/certs.d:/etc/docker/certs.d -v "${{WORKSPACE}}":/go -v /vic-cache:/vic-cache --env-file "vic-internal/{repo}-scenario-{version}-secrets.list" --env-file build.envfile gcr.io/eminent-nation-87317/vic-integration-test:1.46 vic-tools/jenkins/jobs/vic-product-scenario/build.sh ${{TEST_CASES}}
-          ./vic-tools/jenkins/jobs/vic-product-scenario/selenium_grid.sh remove {version}
\ No newline at end of file
+          ./vic-tools/jenkins/jobs/vic-product-scenario/selenium_grid.sh remove {version} {parallel_jobs}
+
+- defaults:
+    name: 'pr-tests'
+    description: 'This is a parameterized build for verifying your test code in a PR'
+    node: '{node}'
+    parameters:
```

Comment:
Is there any way to deduplicate these parameters with the other jobs? It seems like we'd want to keep most of them consistent.